recipe/build.sh

```diff
@@ -18,12 +25,20 @@ do
 	    --disable-overwrite \
 	    --enable-symlinks \
 	    --enable-termcap \
-	    --with-termlib \
```

Comment:
By removing this option, a separate `libtinfo` library will not be built. All of these symbols will be in `libncurses` instead.